Azure/azure-cli
# [AppService] BREAKING CHANGE: az webapp up: add optional params (os & runtime) and updated runtimes

**Description**  
- Add optional parameters (os, runtime) to override az webapp up os & runtime detection
- Updated some webapp runtime values (in WebappRuntimeStacks.json)

**Testing Guide**  

- `az webapp up --resource-group MyRg --name MyWebapp --runtime "node|12-LTS" --os "windows"`

---

This checklist is used to make sure that common guidelines for a pull request are followed.
